conan/tools/apple/apple.py

```diff
@@ -194,26 +204,33 @@ def fix_apple_shared_install_name(conanfile):
     *install_name_tool* utility available in macOS to set ``@rpath``.
     """
 
+    if not is_apple_os(conanfile):
+        return
+
+    xcrun = XCRun(conanfile)
```

Comment:
Just curious (and possibly ignorance about how it works). How this ``XCRun`` connects to the specific ``osxcross`` environment? Via ``os.sdk`` setting? I guess that you have tested against your ``osxcross``, and it works, but as it is not covered by full functional tests, I'd like to understand how it works.

Reply:
I think the xcrun has that relative SDK path hardcoded as the default search location when it was compiled, but I'm not totally sure.

On a semi-related note, I do need to explicitly set `tools.apple:sdk_path` for some CMake logic to work properly - would it be worth it to have the CMakeToolchain tool query XCRun for the SDK path if it's not explicitly configured?